app/javascript/Pages/components/SocialCard.scss

```diff
@@ -0,0 +1,87 @@
+.cardSocial {
+  min-width: 11.75em;
+  height: 16.88em;
+  display: flex;
+  border-radius: 0.6em;
+  margin: 0.5em;
+  overflow: hidden;
+  cursor: pointer;
+  transition: all ease 200ms;
+  position: relative;
+  color: #fff;
+
+  .cardSocial-img {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    z-index: -1;
+  }
+
+  .cardSocial-filter {
+    background-color: rgba(37, 38, 54, 0.4);
+    min-width: 11.75em;
+    height: 16.88em;
+    border-radius: 0.6em;
+    position: relative;
+  }
+
+  .cardSocial-skill-name {
+    left: 50%;
+    top: 50%;
+    width: 90%;
+    transform: translate(-50%, -50%);
+    font-size: 16px;
```

Comment:
**issue (blocking)**: be consistent in the units you use. The general best practice is to use `rem` as it is always related to the font size in the body. If you use `px` they will not be related to anything. If you use `em` they will be related to the font size of the element or the parent.